CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## v0.6
+
+* Added Katz-centrality algorithm (`katz_centrality`) [#366](https://github.com/xgi-org/xgi/issues/366) (@acombretrenouard).
 * Added functions to compute shortest paths (`single_source_shortest_path` and `shortest_path_length`) [#365](https://github.com/xgi-org/xgi/issues/365) (@acombretrenouard)
```

Comment:
v0.6 was already released so these two should be above, for the yet-to-be-named future release

Reply:
Fixed !